apps/omg_watcher/lib/omg_watcher/web/controllers/transaction.ex

```diff
@@ -51,6 +51,7 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
       API.Transaction.submit(tx)
       |> api_response(conn, :submission)
     end
+    |> increment_metrics_counter()
```

Comment:
why abuse with statements and pipes so much? there are better ways to write this

Reply:
for me, shorter is better, and `elixir likes pipes`

Reply:
that's alright, I like pipes as well! but only when they're needed. 
https://github.com/lexmag/elixir-style-guide#needless-pipeline

https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines